spreadsheet/workbook.go

```diff
@@ -746,8 +761,8 @@ func workbookFinalizer(wb *Workbook) {
 // Close closes the workbook, removing any temporary files that might have been
 // created when opening a document.
 func (wb *Workbook) Close() error {
-	if wb.TmpPath != "" && strings.HasPrefix(wb.TmpPath, os.TempDir()) {
-		return os.RemoveAll(wb.TmpPath)
+	if wb.TmpPath != "" {
```

Comment:
If using the disk storage, is there any risk to removing this HasPrefix check?

Reply:
yes, right. Moved this check to tempstorage/diskstore/storage.go